server/src/internalClusterTest/java/org/opensearch/indices/stats/IndexStatsIT.java

```diff
@@ -116,7 +120,23 @@
 
 @ClusterScope(scope = Scope.SUITE, numDataNodes = 2, numClientNodes = 0)
 @SuppressCodecs("*") // requires custom completion format
-public class IndexStatsIT extends OpenSearchIntegTestCase {
+public class IndexStatsIT extends ParameterizedOpenSearchIntegTestCase {
+    public IndexStatsIT(Settings settings) {
+        super(settings);
+    }
+
+    @ParametersFactory
+    public static Collection<Object[]> parameters() {
+        return Arrays.asList(
+            new Object[] { Settings.builder().put(CLUSTER_CONCURRENT_SEGMENT_SEARCH_SETTING.getKey(), false).build() },
+            new Object[] { Settings.builder().put(CLUSTER_CONCURRENT_SEGMENT_SEARCH_SETTING.getKey(), true).build() }
+        );
+    }
+
+    @Override
+    protected Settings featureFlagSettings() {
+        return Settings.builder().put(super.featureFlagSettings()).put(FeatureFlags.CONCURRENT_SEGMENT_SEARCH, "true").build();
+    }
```

Comment:
@harishbhakuni do you mind if I do this as a part of #9622? I want to add some tests specific to concurrent search here.

Reply:
I didn't end up picking this up as a part of the PR above so it still needs parameterization

Reply:
@jed326 sorry for delayed response. just saw this thread.
> I didn't end up picking this up as a part of the PR above so it still needs parameterization

oh okay, sure i will update it as part of this PR.